package.json

```diff
@@ -37,13 +38,15 @@
         "rollup-plugin-json": "4.0.0",
         "rollup-plugin-node-resolve": "~4.2.0",
         "rollup-plugin-sourcemaps": "^0.4.2",
+        "rxjs": "6.4.0",
```

Comment:
It seems like an awkward way to test it. What about either having a test package that we publish just for this purpose. Or maybe it would be better tested with a unit test

Reply:
Good idea. I'll move it to an e2e test which tests the `rm -rf node_modules && yarn install` case.